proto/kvrpcpb.proto

```diff
@@ -226,3 +226,14 @@ message RawDeleteResponse {
     errorpb.Error region_error = 1;
     string error = 2;
 }
+
+message RawScanRequest {
+    Context context = 1;
+    bytes start_key = 2;
+    uint32 limit = 3;
+}
+
+message RawScanResponse {
+    errorpb.Error region_error = 1;
+    repeated KvPair kvs = 2;
```

Comment:
May be we could add the following two element in response:

* `IsTruncated` Specifies whether (true) or not (false) all of the results were returned
* `NextStartKey` indicate next start_key when `IsTruncated` is true.